traffic_ops/app/lib/API/Deliveryservice.pm

```diff
@@ -292,7 +308,10 @@ sub update {
 	}
 	
 	#setting tenant_id to undef if tenant is not set. 
-	my $tenant_id = exists($params->{tenantId}) ? $params->{tenantId} :  undef; 
+	my $tenant_id = exists($params->{tenantId}) ? $params->{tenantId} :  undef;
```

Comment:
Yes it does. This is what we did in the user tenancy as well.
I don't see it as a security hole, as only the user with the proper tenancy can remove the tenancy from the DS.